torch/csrc/jit/passes/tensorexpr_fuser.cpp

```diff
@@ -146,6 +148,13 @@ bool allShapesAreKnown(Value* v) {
   return v->isCompleteTensor();
 }
 
+bool allShapesAreKnown2(Value* v) {
+  if (!v->type()->cast<TensorType>()) {
+    return true;
+  }
+  return v->isCompleteTensor();
```

Comment:
why are we checking that the tensor is complete ? aren't we also expecting tensor types which are profiled but not fully known ?

Reply:
NNC doesn't know how to handle those yet